source/beginners_guide/beginners_guide_index.rst

```diff
@@ -4,3 +4,9 @@
    :glob:
 
    /beginners_guide/raw_beginners_guide
+
+.. toctree::
```

Comment:
The way the beinners guide is structured and the way the dev guide is structured is making it impossible to add it as another sub-point in the sidebar. This does not work